util/gtld_map.go

```diff
@@ -5908,6 +5908,11 @@ var tldMap = map[string]GTLDPeriod{
 		DelegationDate: "2016-07-28",
 		RemovalDate:    "",
 	},
+	"ss": {
+		GTLD:           "ss",
+		DelegationDate: "1985-01-01",
```

Comment:
These TLDs are appearing in: https://data.iana.org/TLD/tlds-alpha-by-domain.txt, but don't yet have delegation data in: https://www.icann.org/resources/registries/gtlds/v2/gtlds.json

So, is this a bug in the ICANN/IANA data, or is there something that `zlint-gtld-update` should be doing differently when delegation data doesn't exist?

Reply:
Ah, or the third option, manually correcting the DelegationDate when `zlint-gtld-update` doesn't have data ( looks like the ICANN json specifically does NOT include ccTLDs ) ?

Reply:
Quoting the zlint-gtld-update pull request:
`Only the first data source provides the delegation and removal dates for TLDs, and is missing ccTLDs like `.uk`, `.ca`, etc. To build a complete `tldMap` the first source is used to build the initial map and any missing TLDs from the alpha-by-domain list are added with an unbounded effective period starting at the date the `.com` gTLD was delegated. This seems like the best compromise since I can't find an equivalent data source for ccTLDs as gTLDs.`